torch/csrc/jit/runtime/static/generated_ops.cpp

```diff
@@ -3437,13 +3437,12 @@ REGISTER_OPERATOR_FUNCTOR(aten::nll_loss, aten_nll_loss, [](Node* n) -> SROperat
       const auto ignore_index = p_node->Input(4).toInt();
       if (p_node->Output(0).isNone()) {
         p_node->Output(0) =
-            at::native::nll_loss(self, target, weight, reduction, ignore_index);
+            at::nll_loss(self, target, weight, reduction, ignore_index);
```

Comment:
This is a pessimization for static runtime. Should apply the BC trick so the old at::native:: is available

Reply:
what's the BC trick?

Reply:
Put the old signature in aten/src/ATen/native/NonSymbolicBC.h and then template the native:: declaration to define both the non-symint and symint simultaneously

Reply:
oh nice. addressed in the reland PR https://github.com/pytorch/pytorch/pull/87095